### PR TITLE
Add frankfurt deployment and service yaml for bedrock-prod

### PR DESCRIPTION
**Pull request by jgmize**
Note that this includes the *names* of secrets, but not the *values* of the secrets themselves.
For #683

### CODE REVIEW

File: apps/bedrock/k8s/frankfurt/bedrock-prod-svc.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    name: bedrock-nodeport
+    namespace: bedrock-prod
+    resourceVersion: "690478"
+    selfLink: /api/v1/namespaces/bedrock-prod/services/bedrock-nodeport
+    uid: a8381c22-52b3-11e7-8dc9-02859405480d
+  spec:
+    clusterIP: 100.67.37.244
+    ports:
+    - name: https
+      nodePort: 30588
```
Comment (metadave) [inline apps/bedrock/k8s/frankfurt/bedrock-prod-svc.yaml]: can the nodeport be left out here to let K8s dynamically set one for us?

File: apps/bedrock/k8s/frankfurt/bedrock-prod-svc.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      app: bedrock-prod
+      heritage: deis
+      router.deis.io/routable: "true"
+    name: bedrock-prod
+    namespace: bedrock-prod
+    resourceVersion: "1710397"
+    selfLink: /api/v1/namespaces/bedrock-prod/services/bedrock-prod
+    uid: a663ca8f-52b3-11e7-8dc9-02859405480d
+  spec:
+    clusterIP: 100.66.12.46
```
Comment (metadave) [inline apps/bedrock/k8s/frankfurt/bedrock-prod-svc.yaml]: do we set this value, or does K8s auto-assign it?

File: apps/bedrock/k8s/frankfurt/bedrock-prod-svc.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      type: web
+    sessionAffinity: None
+    type: NodePort
+  status:
+    loadBalancer: {}
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      router.deis.io/domains: bedrock-prod,www.mozilla.org
```
Comment (metadave) [inline apps/bedrock/k8s/frankfurt/bedrock-prod-svc.yaml]: the deis annotations should eventually be removed, but they're "mostly" harmless for now.

File: apps/bedrock/k8s/frankfurt/bedrock-prod-deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,1302 @@
...
+              cpu: 250m
+              memory: 300Mi
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+        dnsPolicy: ClusterFirst
+        restartPolicy: Always
+        schedulerName: default-scheduler
+        securityContext: {}
+        terminationGracePeriodSeconds: 30
+  status:
```
Comment (metadave) [inline apps/bedrock/k8s/frankfurt/bedrock-prod-deployment.yaml]: I believe the `status` section can be removed.

**Comment by jgmize** [comment]
Commands used to produce the yaml files in this PR:
* [frankfurt](https://github.com/jgmize/dotfiles/blob/0ce2ecb3f663a75158e88bf245cbbffe375f4f5b/.bash_functions.d/k8s#L37)
* [ns](https://github.com/jgmize/dotfiles/blob/0ce2ecb3f663a75158e88bf245cbbffe375f4f5b/.bash_functions.d/k8s#L21) bedrock-prod
* [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) get [deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) -o [yaml](http://yaml.org/) > bedrock-prod-deployments.yaml
* kubectl get [svc](#TODO) -o yaml > bedrock-prod-svc.yaml

**Comment by metadave** [comment]
I'm going to close this for now, we'd most likely need refreshed yaml when we migrate away from Deis Workflow.

**Pull request closed without merging**